package.json

```diff
@@ -63,7 +63,7 @@
     "magicast": "^0.3.3",
     "mlly": "^1.6.1",
     "nuxt": "^3.10.2",
-    "nypm": "^0.3.6",
+    "nypm": "^0.3.8",
```

Comment:
Please avoid unrelated changes in PR (also for lockfile)

Reply:
Sorry, reverted this commit: d619e52290bb43c570579b91c1a7296fc85108d4